libtest/testutils.c

```diff
@@ -212,7 +213,11 @@ assert_gdouble_non_fatal(gdouble actual, gdouble expected, const gchar *error_me
 {
   va_list args;
 
-  if (actual == expected)
+  /* NOTE: inf means infinity i.e. 1/0.00 */
```

Comment:
This comment does not have any useful information/explanation, therefore should not exists. Please remove.